Mathlib/LinearAlgebra/UnitaryGroup.lean

```diff
@@ -55,6 +55,9 @@ inverse.
 abbrev unitaryGroup :=
   unitary (Matrix n n α)
 
+example : Group (unitaryGroup n α) := inferInstance
+example : StarMul (unitaryGroup n α) := inferInstance
```

Comment:
Why?

Reply:
I'm guessing you want more than the comment that is already there?